source/preface_third.md

```diff
@@ -1,10 +1,250 @@
+common situations. We apply warning labels to the recipes, in the hope
+that our students will later do the right tests, even if they have little
+understanding of what those tests mean.  Of course, this is a miserable way to
+learn, and a miserable way to teach.  Our better students see that we are not
+teaching for understanding, but rehearsal, and they suffer the most.  As you
+will soon see, this book is for the teacher as well as the student.  It may be
+no comfort, but we teachers also suffer when we have to teach this way.
+
+Because this is such a common problem, many people have asked why statistics is
+hard to teach, and hard to understand, and they have come to the same insight
+as Simon; our standard statistics use a lot of mathematics, and this
+mathematics is far beyond the level that we can reasonably teach to most of our
+students [@cobb2007introductory].  As Wallis and Roberts
+[-@wallis1957statistics] point out in early textbook, the large majority of our
+students are not fluent in mathematics, and teaching statistics by way of
+mathematics is like teaching rhetoric in a foreign language.
+
+Simon went two steps further than this, although, as you will see, those two
+steps can also be one giant leap.  If he made two steps, the first was to see
+that mathematics was not the only thing that made statistics difficult. He
+realized, with great clarity, that statistical inference --- drawing
+conclusions from noisy data --- involves a lot of hard, clear thinking about
+the data, and the process that made the data.  This hard thinking remains, even
+if you can (you can!) strip out the mathematics.   He writes eloquently about
+this in the introduction of this book, in a section "Why is statistics such
+a difficult subject".   Because statistics is inherently difficult, it is all
+the more important to strip away the extra difficultly that comes from teaching
+through mathematics.  This second, inherent difficulty of statistics is much
+more compelling than the first, incidental and unnecessary difficulty of
+understanding the mathematics, because the inherent difficulty is just that of
+thinking clearly about the problem, and designing a solution.  It is the
+difficultly of thinking logically about data, and noise.
+
+Simon's second step, was to see what he needed to do to bypass the mathematics,
+and go directly to the inherent, interesting problem.  His deep insight was
+this (from the original preface): "Beneath the logic of a statistical inference
+there necessarily lies a physical process".  He saw, remarkably clearly, that
+drawing conclusions from noisy data means *building* a *model* of the noisy
+world, and seeing how that model behaves. That model can be physical, where we
+generate noise with physical devices like dice and spinners and coin-tosses.
+In fact, Simon used and taught exactly these kinds of devices in his first
+experiments in teaching these methods [@simon1969basic].  He very quickly
```

Comment:
This is also one of the methods that Maxwell used to figure out the equations of electromagnetics (he envisioned physical models for mathematical operations; see Wikipedia and https://www.cc.gatech.edu/aimosaic/faculty/nersessian/papers/maxwell-and-the-method-of-physical-analogy.pdf).